pages/affiliated.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: page
+title: Affiliated Packages
+permalink: /affiliated
+nav_order: 6
+---
+
+
+Affiliated Packages {#affiliated_packages}
+====================
+
+In the following, projects that work closely together with scikit-hep are described. They extend the Python ecosystem in HEP and remain, due to their size and scope, generally independent of scikit-hep.
```

Comment:
You should use "Scikit-HEP" for the project (2 occurrences in these 2 lines).

Reply:
done